frontend/index.html

```diff
@@ -5,27 +5,86 @@
+        <link rel="stylesheet" href="http://ajax.googleapis.com/ajax/libs/angular_material/1.0.0/angular-material.min.css">
     </head>
-    <body >
-    <header><h1>AZC MAP</h1></header>
-        <section id="main-box">
-            <div ng-controller="MapCtrl as MapCtrl" id="main-map">
-            <ui-gmap-google-map center='map.center' options="MapCtrl.options" zoom='map.zoom'></ui-gmap-google-map>
+    <body ng-controller="MapCtrl as MapCtrl">
 
-            <script src="bower_components/angular/angular.min.js"></script>
 
-            <script src="bower_components/angular-google-maps/dist/angular-google-maps.min.js"></script>
+        <section >
+            <div ng-controller="AppCtrl as AppCtrl" id="main-box">
+                <md-toolbar style="position: absolute; z-index: 50; background-color: rgb(255, 82, 82);">
+                  <div class="md-toolbar-tools">
+                    <md-button class="md-icon-button" aria-label="Settings">
+                      <md-icon md-svg-icon="assets/img/menu.svg" ></md-icon>
+                    </md-button>
+                    <h2>
+                      <span>MAPA DE CASOS - AEDES AEGYPT</span>
+                    </h2>
+                    <span flex></span>
+                    <md-button class="md-icon-button" aria-label="Favorite">
+                      <md-icon md-svg-icon="assets/img/favorite.svg"></md-icon>
+                    </md-button>
+                    <md-button class="md-icon-button" aria-label="More">
+                      <md-icon md-svg-icon="assets/img/more_vert.svg"></md-icon>
+                    </md-button>
+                  </div>
+                </md-toolbar>
+               
 
-            <script src="bower_components/angular-simple-logger/dist/angular-simple-logger.min.js"></script>
 
-            <script src="bower_components/lodash/dist/lodash.min.js"></script>
 
-            <script src="assets/js/app.js"></script>
-        </div>
-        <div id="main-sidebar">
-            <a href="">Testando 1 2 3</a>
-        </div>
+<!--                 <div ng-app="myapp">
+                    <div layout="row" ng-controller="sidenavCtrl">
+                        <md-sidenav md-component-id="left" md-is-open="isSidenavOpen" class="md-sidenav-left">
+                            Left Nav!
+                        </md-sidenav>
+                         <md-content>
+                          </md-content>
+                    </div>
+                </div>
+
+ -->
+
+                <md-button class="md-fab md-fab-bottom-right" aria-label="Add" ng-click="showAdd($event)"><i class="fa fa-plus"></i>
+                    <ng-md-icon icon="add"></ng-md-icon>
+                </md-button>
+                <ui-gmap-google-map center='map.center' options="MapCtrl.options" zoom='map.zoom'></ui-gmap-google-map>
+            </div>
+            
         </section>
+
+
+
+
+<script type="text/javascript">
+    var app = angular.module('myapp', ['ngMaterial']);
```

Comment:
Why is it here? The module creation is under app.js file. This line should be removed.